src/tests/ftest/daos_test/dfuse.py

```diff
@@ -88,7 +85,7 @@ def test_daos_dfuse_unit(self):
         daos_test_env = cmocka_utils.get_cmocka_env()
         intercept = self.params.get('use_intercept', '/run/intercept/*', default=False)
         if intercept:
-            daos_test_env['LD_PRELOAD'] = os.path.join(self.prefix, 'lib64', 'libioil.so')
+            daos_test_env['LD_PRELOAD'] = os.path.join(self.prefix, 'lib64', il_lib)
             daos_test_env['D_LOG_FILE'] = get_log_file('daos-il.log')
```

Comment:
It's probably best to rename this as well.

Reply:
I see. Thank you very much for your clarification! I will fix it.